docs/tutorials/command_line_client.md

```diff
@@ -74,7 +74,8 @@ synapse [-h] [--version] [-u SYNAPSEUSER] [-p SYNAPSEPASSWORD] [-c CONFIGPATH] [
 - [create](#create): Creates folders or projects on Synapse
 - [store-table](#store-table): Creates a Synapse Table given a csv
 - [onweb](#onweb): opens Synapse website for Entity
-- [login](#login): login to Synapse and (optionally) cache credentials
+- [login](#login): Verify credentials can be used to login to Synapse.
```

Comment:
This makes me think, is there a use for the "log-out" command any longer?

Reply:
There is no "log-out" CLI command

Reply:
Ah - my mistake - I think there is a `logout` function in client.py.

Reply:
I'm fine leaving this in the client. It's not really useful, but not harmful either.